velox/functions/prestosql/types/TDigestType.h

```diff
@@ -73,8 +74,10 @@ using TDigest = CustomType<TDigestT>;
 
 class TDigestTypeFactories : public CustomTypeFactories {
  public:
-  TypePtr getType() const override {
-    return TDIGEST();
+  TypePtr getType(const std::vector<TypeParameter>& parameters) const override {
+    VELOX_CHECK_EQ(parameters.size(), 1);
+    VELOX_CHECK(parameters[0].kind == TypeParameterKind::kType);
```

Comment:
can we use VELOX_CHECK_EQ?

Reply:
No there is a build error on GitHub complaining don't know how to convert `kType` to string